.eslintrc

```diff
@@ -8,34 +23,48 @@
     "eslint:recommended"
   ],
   "rules": {
+    "react/jsx-uses-react": 1,
+    "react/jsx-uses-vars": 1,
+
     "mozilla/components-imports": 1,
     "mozilla/import-globals-from": 1,
     "mozilla/this-top-level-scope": 1,
 
-    "react/jsx-uses-react": 1,
-
     "array-bracket-spacing": [2, "never"],
+    "camelcase": 0,
     "comma-dangle": 0,
     "computed-property-spacing": [2, "never"],
+    "default-case": 0,
+    "eqeqeq": 1,
+    "func-names": 0,
+    "func-style": 0,
+    "global-require": 0,
+    "id-blacklist": 0,
+    "id-length": 0,
+    "id-match": 0,
+    "init-declarations": 0,
+    "max-len": 0,
+    "max-params": 0,
+    "newline-after-var": 0,
+    "no-bitwise": 0,
     "no-console": 1,
+    "no-empty-function": 0,
+    "no-inline-comments": 0,
+    "no-invalid-this": 0,
+    "no-magic-numbers": 0,
+    "no-negated-condition": 0,
+    "no-shadow": 1, // TODO: Remove this?
     "no-trailing-spaces": [2, {"skipBlankLines": false}],
     "no-undef": 2,
-    "no-unused-vars": 2,
+    "no-underscore-dangle": 0,
+    "no-use-before-define": [1, {"functions": false, "classes": false}],
+    "no-unused-vars": [2, {"vars": "all", "args": "none"}],
+    "no-warning-comments": 1,
     "object-curly-spacing": [2, "never"],
+    "prefer-reflect": 0,
```

Comment:
Turns out adding `prefer-const: 1` throws 204 warnings, so it will be mildly disruptive.